sql-imap-test/src/main/java/com/hazelcast/jet/tests/sql/tests/AbstractSqlTest.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.tests.sql.tests;
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.jet.tests.common.AbstractSoakTest;
+import com.hazelcast.jet.tests.sql.pojo.Key;
+import com.hazelcast.jet.tests.sql.pojo.Pojo;
+import com.hazelcast.map.IMap;
+import com.hazelcast.sql.SqlResult;
+import com.hazelcast.sql.SqlService;
+import org.junit.Assert;
```

Comment:
Rather use assert methods from `AbstractSoakTest`, otherwise you need to put junit on server classpath.

Reply:
Done